README.md

```diff
@@ -59,6 +60,8 @@ If using a collection to produce `<select>` `<option>`s, the following may also
 - `[textAttribute]`: model attribute to use as the text of the option node in the select box
 - `[yieldModel]`: [default: `true`] if options is a collection, yields the full model rather than just its `idAttribute` to `.value`
 
+When the collection changes, the view will try and maintain its currently `.value`.  If the corresponding model is removed, the <select> control will default to the 0th index <option> and update its value accordingly.
```

Comment:
hmm what are you using as a previewer? my markdown previewer looks ok:

![screen shot 2015-04-15 at 11 37 46 am](https://cloud.githubusercontent.com/assets/1003261/7166639/edb6545c-e363-11e4-8c49-ac22e63e85a5.png)